.github/workflows/deploy.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     - uses: actions/checkout@v3
     - uses: actions/setup-node@v3
       with:
-        node-version: 16
+        node-version: 18

```

Comment:
Is this required in order for this to work?

Reply:
Shouldn't be, I just wanted to align with the node version mentioned in your Dockerfile